src/deviceApiCalls/__generated__/validators-ts.d.ts

```diff
@@ -18,6 +99,143 @@ export interface AutofillFeatureToggles {
   password_generation?: boolean;
   credentials_saving?: boolean;
   inlineIcon_credentials?: boolean;
+  third_party_credentials_provider?: boolean;
+}
+
+// availableInputTypes.json
+
+/**
+ * For each main autofill types, it maps specific fields to their availability
+ */
+export interface AvailableInputTypes {
+  /**
+   * maps field types and the availability of data for the current site
+   */
+  credentials?: {
+    username?: boolean;
+    password?: boolean;
+    [k: string]: unknown;
```

Comment:
This occurs when the schema is left 'open' on an object - you need to add `additionalProperties: false` , that is unless you want this behaviour?